app/main/posts/modify/post-edit.controller.js

```diff
@@ -36,7 +36,7 @@ function (
         });
 
         if (post.form) {
-            $scope.form = FormEndpoint.get({ id: post.form.id }, function (form) {
+            $scope.form = FormEndpoint.getFresh({ id: post.form.id }, function (form) {
```

Comment:
This won't use the cache at all, which might be fine but it will probably dramatically increase requests.